ue4docker/dockerfiles/ue4-build-prerequisites/windows/Dockerfile

```diff
@@ -35,10 +39,22 @@ RUN start /wait %TEMP%\directx_redist.exe /Q /T:%TEMP% && `
 	expand %TEMP%\Jun2010_XAudio_x64.cab -F:XAPOFX1_5.dll C:\GatheredDlls\ && `
 	expand %TEMP%\Jun2010_XAudio_x64.cab -F:XAudio2_7.dll C:\GatheredDlls\
 
+# Retrieve the DirectX shader compiler files needed for DirectX Raytracing (DXR)
+RUN curl --progress -L "https://github.com/microsoft/DirectXShaderCompiler/releases/download/v1.6.2104/dxc_2021_04-20.zip" --output %TEMP%\dxc.zip && `
+	powershell -Command "Expand-Archive -Path \"$env:TEMP\dxc.zip\" -DestinationPath $env:TEMP" && `
+	xcopy /y %TEMP%\bin\x64\dxcompiler.dll C:\GatheredDlls\ && `
+	xcopy /y %TEMP%\bin\x64\dxil.dll C:\GatheredDlls\
+
 # Gather the Vulkan runtime library
 RUN curl --progress-bar -L "https://sdk.lunarg.com/sdk/download/latest/windows/vulkan-runtime-components.zip?u=" --output %TEMP%\vulkan-runtime-components.zip
 RUN 7z e %TEMP%\vulkan-runtime-components.zip -oC:\GatheredDlls -y "*\x64\vulkan-1.dll"
 
+# Verify that the DLL files copied from the host can be loaded by the container OS
```

Comment:
I moved verification step to a later stage so that it covers DirectX and Vulkan libs.